packages/paste-core/components/form-pill-group/stories/customization.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import {useTheme} from '@twilio-paste/theme';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {useFormPillState, FormPillGroup, FormPill} from '../src';
+
+export const CustomFormPillGroup: React.FC = () => {
```

Comment:
nit(non-blocking): we can and probably should start all using the `Story` type from `@storybook/react` to type these story functions going forward.